app/app.go

```diff
@@ -9,6 +9,7 @@ import (
 	"os"
```

Comment:
~General comment, need to do something if the packet finalized already?~

https://github.com/dymensionxyz/dymension/blob/4ab9dc94758ca1f28d18679bd379dcc96053a9a1/x/delayedack/ibc_middleware.go#L83-L86

Reply:
I think it's fine